src/modules/client/pages/ni/customers/CustomersDirectory.vue

```diff
@@ -193,26 +193,26 @@ export default {
     },
     async getCustomers () {
       try {
-        this.tableLoading = true;
-        this.customers = await Customers.list();
+        return Customers.list();
       } catch (e) {
-        this.customers = [];
         console.error(e);
-      } finally {
-        this.tableLoading = false;
       }
     },
     async getCustomersFirstIntervention () {
       try {
-        this.firstInterventions = await Customers.listWithFirstIntervention();
-        this.customers = this.customers.map(customer => ({
+        this.tableLoading = true;
+        this.firstInterventions = Object.freeze(await Customers.listWithFirstIntervention());
+        const customers = await this.getCustomers();
```

Comment:
tu peux mettre directement `const customers = await Customers.list();`
j'ai l'impression que la methode `getCustomers` n'a plus trop d'utilité du coup
surtout que `this.getCustomers` est appelé dans le `refresh` mais cet appel ne sert a rien et du coup il est fait 2 fois

et pourras tu changer le `mounted` en `created` ?